lib/ts/recipe/multifactorauth/api/implementation.ts

```diff
@@ -21,7 +21,7 @@ import SessionError from "../../session/error";
 
 export default function getAPIInterface(): APIInterface {
     return {
-        updateSessionAndFetchMfaInfoPUT: async ({ options, session, userContext }) => {
+        resyncSessionAndFetchMFAInfoPUT: async ({ options, session, userContext }) => {
```

Comment:
Why was this renamed? I'm not completely against it but I like update a bit better because it matches with `updateAccessTokenPayload`.